### ISSUE TITLE
access: Approver Access ("approvers")

### ISSUE THREAD

**Issue by oiooj** [golang/go#43336]
Gerrit email: bk@golangcn.org
https://go-review.googlesource.com/q/owner:bk%2540golangcn.org

Approver Access ("approvers")

https://go-review.googlesource.com/#/admin/groups/1005,members

Hi, @dmitshur I have been around for a long time (from `vgo`), and made many commits.  After #40699, I think I need apply a `Trust +1` votes to simplify my future work.

/cc @toothrot @rsc

**Comment by oiooj** [comment]
ping

**Comment by ianlancetaylor** [comment]
CC @dmitshur @toothrot @cagedmantis

**Comment by toothrot** [comment]
Hi @oiooj. I apologize for the delay. Granted, and thanks for your contributions!
